Sources/StytchCore/StytchClientCommon/Models/StytchError.swift

```diff
@@ -1,118 +1,131 @@
 import Foundation
 
-/// A type representing an error within the Stytch ecosystem.
-public struct StytchError: Error, Decodable, Equatable {
+/// Base class representing an error within the Stytch ecosystem.
+public class StytchError: Error, Decodable {
     private enum CodingKeys: CodingKey {
-        case statusCode
-        case requestId
-        case errorType
-        case errorMessage
-        case errorUrl
+        case name
+        case description
+        case url
```

Comment:
I think we want to make the same change here that we did for JS and Android: move URL to a property of the type that specifically has it (SDKAPIError, IIRC)